docs/pages/setup/admin/daemon.mdx

```diff
@@ -31,117 +35,174 @@ $ readlink /sbin/init
+If you have not already, select and complete the appropriate installation instructions for your environment.
 
 (!docs/pages/includes/install-linux.mdx!)
 
-Teleport requires a configuration file to run. After installation, no
-configuration file exists. We will create a minimal configuration file to show
-you how to run Teleport as a daemon:
+Teleport requires a [configuration yaml file](https://goteleport.com/docs/setup/reference/config/) to be created and configured in order to run. After initial installation, no
+configuration file exists until it is manually created by a user. This guide will use a minimal teleport configuration based on defaults, however in production environments we
+recommend [reviewing configuration file options](https://goteleport.com/docs/setup/reference/config/) to better suit your own specific needs.
+
+To create the default teleport configuration file, enter the following command:
 
 ```code
 $ sudo teleport configure -o file
 Wrote config to file "/etc/teleport.yaml". Now you can start the server. Happy Teleporting!
 ```
 
-## Step 2/3. Create a systemd unit file
+## Creating and Configuring the systemd Daemon
+
+Once you've initiially installed the `teleport` binary and created a Teleport configuration file, the Teleport daemon must be created and configured. While a daemon can be created and configured manually, the
+Teleport binary includes tooling to assist users in automatically creating the necessary configuration, and package managers may include a good daemon configuration by default.
+
+Systemd is designed to use [unit files](https://www.freedesktop.org/software/systemd/man/systemd.unit.html) which are the primary file used to define and set the conditions for
+any needed service daemon on systemd, in this case the `teleport` service. The rest of this guide will focus on creating and configuring the teleport service daemon depending on your preferred installation method, either a package manager, or from source.
+
+### Creating the systemd Daemon following Package Manager Installation
+
+If Teleport is being installed using a DEB or RPM package manager, a `teleport` systemd daemon configuration will have been included as part of the teleport package installation by default. To check that this daemon was installed correctly,
+enter the following command:
+
+```code
+$ sudo systemctl status teleport
+```
+
+Users will see output similar to the following, including the file path (`/lib/systemd/system/teleport.service`) that contains the unit file for the systemd configuration being applied:
+
+```code
+● teleport.service - Teleport SSH Service
+     Loaded: loaded (/lib/systemd/system/teleport.service; disabled; vendor preset: enabled)
+     Active: inactive (dead)
+```
 
-Copy the recommended Teleport unit file for `systemd` below and paste the
-contents into a file called `/usr/lib/systemd/system/teleport.service`:
+The contents of the default unit file will reflect the following defaults:
```

Comment:
Do you think we need to put the contents of the file here?

One of the reasons for this new command is to reduce the number of duplicated systemd examples we have throughout the code and the docs. If we list the contents here, we have to keep it in sync with future changes.

Maybe instead we could encourage the user to run `teleport install systemd` without the `-o` flag if they want to inspect the default unit file.

Reply:
This is a really good point in keeping things evergreen, I don't see why the contents of the unit file would be necessary in this document and removed it.